tdrs-backend/docker-compose.yml

```diff
@@ -42,19 +42,19 @@ services:
   web:
     restart: always
     environment:
-      - DB_USER=tdpuser
-      - DB_PASSWORD=something_secure
+      - AV_SCAN_URL=http://clamav-rest:9000/scan
+      - DB_HOST=postgres
       - DB_NAME=tdrs_test
+      - DB_PASSWORD=something_secure
       - DB_PORT=5432
-      - DB_HOST=postgres
-      - DJANGO_SETTINGS_MODULE=tdpservice.settings.local
-      - DJANGO_CONFIGURATION=Local
-      - AV_SCAN_URL=http://clamav-rest:9000/scan
+      - DB_USER=tdpuser
+      - DJANGO_CONFIGURATION=${DJANGO_CONFIGURATION:-Local}
+      - DJANGO_SETTINGS_MODULE=${DJANGO_SETTINGS_MODULE:-tdpservice.settings.local}
```

Comment:
This syntax allows us to declare defaults in the base compose file but still allows setting a differing value in the `.env` file if a developer needs to override the settings module in use for local testing.